packages/next/build/webpack-config.ts

```diff
@@ -1451,7 +1451,12 @@ export default async function getBaseWebpackConfig(
         new MiddlewarePlugin({ dev, webServerRuntime }),
       process.env.ENABLE_FILE_SYSTEM_API === '1' &&
         webServerRuntime &&
-        new FunctionsManifestPlugin({ dev, webServerRuntime }),
+        new FunctionsManifestPlugin({
```

Comment:
Might be unrelated to this PR but why are we flagging this behind `ENABLE_FILE_SYSTEM_API`, won't we want this manifest besides then?

Reply:
I got from [filesystem api doc](https://vercel.com/docs/file-system-api), so far it's beta and will be enabled when this flag set to `1` in env. So I picked this one